StrataMain.lean

```diff
@@ -205,10 +205,9 @@ def pyAnalyzeCommand : Command where
     if verbose then
       IO.println "Inlined: "
       IO.print newPgm
+    let solverName : String := "StrataTest/Languages/Python/z3_parallel.py"
```

Comment:
I think that any file that is referenced in StrataMain should have its place in Strata, in StrataTest I think. Can you put it into `Strata/Languages/Python` perhaps?